Tools/autotest/param_metadata/param_parse.py

```diff
@@ -1,14 +1,18 @@
 #!/usr/bin/env python
 
-import os, glob, re, sys
+import glob
+import os
+import re
+import sys
+from optparse import OptionParser
 
-from param import *
-from wikiemit import WikiEmit
-from xmlemit import XmlEmit
 from htmlemit import HtmlEmit
```

Comment:
I get that this is ordered alphabetically but it makes more sense that _htmlemit_ is together with the other *Emit*s